### PR TITLE
On the Road to CDI Compatibility

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
I don't know exactly who I should ask for review when it comes to editorial standards or so. All feedback is welcome! :-)

**Review by gsmet — COMMENTED** [review]
I added a small comment.

The tone is refreshing and I like it. Let's see what others think.

File: _posts/2023-06-30-on-the-road-to-cdi-compatibility.adoc [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: 'On the Road to CDI Compatibility'
+date: 2023-06-30
+tags: arc cdi
+synopsis: 'Quarkus to become a CDI Lite compatible implementation'
+author: lthon
+---
+
+Ever since the very first days of Quarkus, the days that are now covered by the blissful fog of oblivion, the days that the survivors only talk about after a few pints of beer, the days that when remembered spark bright protean shamrocks in the middle of the night, dependency injection container was an integral part of the envisioned framework.
```
[the remark this answers is not in the document]
Comment (Ladicek) [reply _posts/2023-06-30-on-the-road-to-cdi-compatibility.adoc]: Fixed.

**Review by mkouba — APPROVED** [review]
I think that it's really great!

**Comment by Ladicek** [comment]
Rebased, reworded to reflect the fact that Quarkus 3.2 is already out, and scheduled for July 12.

I hope we can get some more review until then, but if not, I'll merge on July 11 :-)

**Comment by Ladicek** [comment]
Merging per above.

**Comment by tqvarnst** [comment]
I checked the post before, and while it's too late to review, I just want to say good work @Ladicek . I think it's clear and fresh.

**Comment by Ladicek** [comment]
Thank you!

**Comment by holly-cummins** [comment]
Same! These 'behind the scenes' stories are always great assets to have, and this one is both useful and adorable. 🥇

**Pull request merged**